common/services/contentful.ts

```diff
@@ -87,22 +87,37 @@ const service = {
     }
   },
   fetchEntries: async () => {
-    const entries = await service.client.getEntries()
+    const entries = await service.client.getEntries({
+      content_type: 'whatsNew',
+    })
 
     if (!entries.items?.length) {
       return []
     }
 
     return entries.items.map(
-      ({ fields: { title, body, briefBannerText, date } }) => ({
+      ({ fields: { title, body, briefBannerText, date } }: {fields: { title: string, body: string, briefBannerText: string, date: any}}) => ({
         title,
         body,
         briefBannerText,
         date: new Date(date),
       })
     )
   },
-  convertToHTMLFormat: body => documentToHtmlString(body, options),
-  formatDate: date => format(date, DATE_FORMATS.WITH_MONTH),
+  fetchEntryBySlugId: async (slugId: any) => {
+    const entry = await service.client.getEntries({
+      content_type: 'dedicatedContent',
+      'fields.slug[in]': slugId,
+    })
+
+    if (!entry.items?.length) {
+      return []
```

Comment:
I feel like this should be `return undefined` so that the function signature is `Object?` instead of `Object | Array`